### ISSUE TITLE
`@golevelup/nestjs-rabbitmq`5.6.0 breaks connection to AmazonMQ RabbitMQ

### ISSUE THREAD

**Issue by Vovcharaa** [golevelup/nestjs#876]
AmazonMQ RabbitMQ uses `amqps` by default.
This breaks apps because latest published version disallows `amqps` protocol and fix is not published yet.
Can you release fix asap? And possibly retract `5.6.0` version to avoid usage of broken package.

**Comment by mozhan1** [comment]
in the latest pr this line got added

```
const rabbitMQRegex =
   /^amqp:\/\/(([^:]+):([^@]+)@)?([^:/]+)(:[0-9]+)?(\/[^\/]+)?$/;

```
the problem is it does not accept amqps connections, thus it is failing when using amqps.

**Comment by Vovcharaa** [comment]
@mozhan1 It was already fixed in `master` branch yesterday but fix hasn't been released yet.

**Comment by underfisk** [comment]
@WonderPanda Are you able to publish a new version to address this issue?

**Comment by WonderPanda** [comment]
Yeah this is published and available now as @golevelup/nestjs-rabbitmq@5.6.1
